website/docs/getting_started/analysis_options.tsx

```diff
@@ -0,0 +1,12 @@
+const raw = `
+analyzer:
+  plugins:
+    - custom_lint
+`;
+
+export default {
+  raw: raw,
+  hooks: raw,
+  codegen: raw,
+  hooksCodegen: raw,
+};
```

Comment:
I honestly wasn't sure what to put here and just took inspiration from [dart_pubspec.tsx](https://github.com/rrousselGit/riverpod/blob/docs-v2/website/docs/getting_started/dart_pubspec.tsx) and [pubspec.tsx](https://github.com/rrousselGit/riverpod/blob/docs-v2/website/docs/getting_started/pubspec.tsx).